data/ka'het/ka'het.txt

```diff
@@ -73,6 +73,455 @@ phrase "ka'mar"
 	word
 		"'mar"
 
+# Hails:
+phrase "friendly ka'het"
+	word
+		"Current objective: "
+	word
+		"aluminum"
+		"copper"
+		"gold"
+		"iron"
+		"lead"
+		"neodymium"
+		"platinum"
+		"silicon"
+		"silver"
+		"titanium"
+		"tungsten"
+		"uranium"
+	word
+		" "
+	phrase
+		"ka'het mining"
+		"ka'het hauling"
+
+phrase "ka'het mining"
+	word
+		"gathering"
+		"mining"
+		"acquisition"
+		"obtainment"
+		"attainment"
+		"recovery"
+	word
+		". "
+	word
+		"Research state: seeking for suitable asteroids."
+
+phrase "ka'het hauling"
+	word
+		"hauling"
+		"freighting"
+		"transfer"
+		"relocation"
+		"allocation"
+	word
+		". Resource destination: station "
+	word
+		"Seli"
+		"Parii"
+	word
+		", area "
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		"."
+
+phrase "friendly ka'het"
+	word
+		"Primary systems check: "
+	word
+		"shield matrices"
+		"hyperdrive"
+		"life support systems"
+		"energy generator"
+		"weapon systems"
+		"main thrusters"
+		"main steering"
+		"main coolers"
+		"Ravager emitters"
+		"Annihilator loading systems"
+		"tracking systems"
+	word
+		": online. Secondary systems check: "
+	word
+		"solar wind harvester: online and active"
+		"energy storage: available"
+		"support coolers: online and active"
+		"shield restorers: online and active"
+		"long range antennae: available"
+	word
+		"."
+
+phrase "friendly ka'het"
+	word
+		"Current objective: systems "
+	word
+		"patrol"
+		"garrison"
+		"vigilance"
+	word
+		". Threats identified: none. Estimated progress: "
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		"%."
+
+phrase "friendly ka'het"
+	word
+		"Het status: "
+	word
+		"operational"
+		"active"
+		"alive"
+		"functioning"
+		"awake"
+	word
+		". Expected lifetime: "
+	word
+		""
+		"1"
+		"2"
+		"3"
+		"3"
+		"3"
+		"4"
+		"4"
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		" years."
+
+phrase "friendly ka'het"
+	word
+		"Current"
+		"Selected"
+		"Chosen"
+		"Main"
+		"Primary"
+	word
+		" target: <ship>. Government: "
```

Comment:
Does this actually work?

Reply:
Absolutely, see the comments for screenshots